src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSourceContext.java

```diff
@@ -147,7 +151,13 @@ public final boolean wantForkPRs() {
     public final Set<ChangeRequestCheckoutStrategy> forkPRStrategies() {
         return forkPRStrategies;
     }
-
+    /**
+     * Returns the strategy used to notify Github of build status.
+     * @return the strategy used to notify Github of build status.
+     */
```

Comment:
put in a `@since TODO` javadoc comment on all new methods and classes. The changes to this class look good